code/data_structures/src/tree/binary_tree/binary_tree/traversal/postorder/postorder.py

```diff
@@ -0,0 +1,25 @@
+# Part of Cosmos by OpenGenus Foundation
+class node:
+    def __init__(self, value):
+        self.data = value
+        self.left = None
+        self.right = None
+
+
+def post_order(root):
+    if root:
```

Comment:
Use `if is not root:` to compare with None.

Reply:
[reference](https://stackoverflow.com/questions/2209755/python-operation-vs-is-not)